ReviewChecklist.md

```diff
@@ -7,6 +7,12 @@ and solve common issues before they are pointed out in the review.
 It is also meant to serve as a final checklist for reviewers to go through before approving a PR.
 
 ## Before You Submit a PR
+- [ ] **Do you have any other open PRs?**
+    Work on a PR is not done until it is merged or closed.
+    Our reviewing capacity is limited, so we require that external contributors work on **no more than one PR at a time**.
+    - If your PR is not getting reviewed, feel free to bring it to our attention on the [#solidity-dev](https://gitter.im/ethereum/solidity-dev) channel.
+    - Unless they were requested, we are going to close any excess PRs, leaving only the earliest one open.
+        You may reopen them, one at a time, when your current PR is done.
 - [ ] **Is the issue ready to be worked on?**
     - If the issue does not have a desirability label (`nice to have`, `should have`,
         `must have eventually`, `must have`, `roadmap`) we have not yet decided whether to implement it.
```

Comment:
```suggestion
    - If the issue does not have a desirability label (`nice to have`, `should have`,
        `must have eventually`, `must have`, `roadmap`) we have not yet decided whether to implement it.
```
- I would remove the **must have eventually** (it's meaningless).
- What is the difference between **should have** and **must have**? Wouldn't it be simpler to just categorize as what we must have and not have (even if not right now)?

Reply:
Marking as resolved since this is unrelated to the PR and we discussed it on the chat.